polkadot/node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -1375,6 +1381,13 @@ impl Initialized {
 			}
 			for (validator_index, (kind, _sig)) in new_state.votes().valid.raw() {
 				let is_backer = kind.is_backing();
+				gum::trace!(
```

Comment:
Same, I think at least debug should be fine here.

Reply:
Bumped